src/vuestic-theme/vuestic-components/va-select/VaSelect.demo.vue

```diff
@@ -79,6 +79,13 @@
         :options="defaultSelect.options"
       />
     </VbCard>
+    <VbCard title="No clear" style="width: 400px;">
+      <va-select
+        v-model="defaultSelect.value"
+        :options="defaultSelect.options"
+        no-clear
```

Comment:
maybe, it's better to name this clearable and make it true by default.
In any case, add a new prop to the doc

Reply:
Everywhere I've seen (quasar, vuetify), `clearable` is false by default.